app/views/shared/_cash_transactions.html.erb

```diff
@@ -0,0 +1,32 @@
+<% @cash_transaction_types.each do |type| %>
+
+  <div class="govuk-!-padding-bottom-4"></div>
+
+  <h2 class="govuk-heading-m"><%= t('.cash_heading') %> <%= t(".#{type}") %></h2>
+
+  <%= govuk_hint t('.cash_hint') %>
+
+  <table class="govuk-table" style="margin-bottom:0px;">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header" scope="col"><%= t('.month') %></th>
+      <th class="govuk-table__header govuk-table__cell--numeric govuk-!-font-weight-bold" scope="col"><%= t('.amount') %></th>
+    </tr>
+    </thead>
+
+    <tbody class="govuk-table__body">
+    <% @cash_transactions.select {|i| i.transaction_type.name == type}.each do |transaction| %>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell"><%= t('.dates', date_end: l(transaction.transaction_date.at_end_of_month , format: :short_date), date_start: l(transaction.transaction_date.at_beginning_of_month, format: :short_date)) %></td>
+        <td class="govuk-table__cell govuk-table__cell--numeric"><%= transaction.amount %></td>
```

Comment:
number_to_currency should be added here. e.g. `<%= number_to_currency(transaction.amount) %>`